features/theme.feature

```diff
@@ -74,7 +74,7 @@ Feature: Manage WordPress themes
   Scenario: Install a theme, activate, then force install an older version of the theme
     Given a WP install
 
-    When I run `wp theme install p2 --version=1.4.2`
+    When I run `wp theme install twentytwelve --version=1.4.2`
     Then STDOUT should not be empty
 
     When I run `wp theme list --name=p2 --field=update_version`
```

Comment:
The following steps all need to be updated as well because we're no longer installing p2